subgraphs/_reference_/src/sdk/protocols/bridge/index.ts

```diff
@@ -85,29 +31,40 @@ export class SDK {
     this.Protocol.sdk = this;
   }
 
-  static initialize<T>(
```

Comment:
Being purists, we should leave this to not break old implementations that depend on it.

If you want to ensure that no one uses it in the future, and to enforce people to upgrade without breaking their build, you can add a deprecated note. The linter should warn you if you still use it:

https://jsdoc.app/tags-deprecated.html

Reply:
Where are you storing the version of the SDK so that I can reference it in the deprecated comment?

Reply:
Nowhere yet. SDK versioning is something we should consider adding.
It is fine if you don’t mention a version number for now, a link to the PR might suffice

Reply:
Added link